share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -762,6 +676,28 @@ ml-linux-x86_64-rocm-build:
     - artifacts: True
       job: ml-linux-x86_64-rocm-generate
 
+########################################
+# Machine Learning - Darwin aarch64 (MPS)
```

Comment:
Yes, that's why I removed it from this PR, because each stack concretizes to ~300 packages, and I'm not sure our current mac runners can handle up to ~600 package builds per PR potentially without proper sharing of binaries.